Components/App.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { ChangeEvent, useState, FC } from 'react';
+import { ITask } from './interfaces/ITask';
+import TodoTask from './ToDoTask/ToDoTask';
+import CustomDiv from './CustomDiv/CustomDiv';
+import CustomButton from './CustomButton/CustomButton';
+import CustomInput from './CustomInput/CustomInput';
+
+const App: FC = () => {
+  const [task, setTask] = useState<string>("");
+  const [deadline, setDeadline] = useState<number>(0);
+  const [todoList, setTodoList] = useState<ITask[]>([]);
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>): void => {
+    if (event.target.name === "task") {
+      setTask(event.target.value);
+    } else {
+      setDeadline(Number(event.target.value));
+    }
+  };
+
+  const addTask = (): void => {
+    const newTask = { taskName: task, deadline: deadline };
+    setTodoList([...todoList, newTask]);
+    setTask("");
+    setDeadline(0);
+  };
+
+  const completeTask = (taskNameToDelete: string): void => {
+    setTodoList(todoList.filter((task) => {
+      return task.taskName !== taskNameToDelete;
+    }));
+  };
+  const sameStyles = {
+    display:'flex',
+    justifyContent: 'center',
+    alignItems: 'center'
+  }
+
+  return (
+    <CustomDiv style={{ width:'110%'}}>
+      <CustomDiv style={{ 
+        ...sameStyles,
+        backgroundColor:'lightseagreen',
+        fontStyle: 'oblique',
+        color: 'black',
+        fontSize:'50px',
+        marginRight: '111px',
+        marginLeft: '-10px'
+       }}>To Do List Using TypeScript</CustomDiv>
+      <CustomDiv style={{
+        display: 'flex',
+        alignItems: 'center',
+        flexDirection: 'column',
+        width: '100vw',
+        height: '100vh',
+        fontFamily: 'Arial, Helvetica, sans-serif'
+      }}>
+        <CustomDiv style={{
+          ...sameStyles,
```

Comment:
this modification should be inside CustomDiv only

Reply:
it is not done bro ba3da mawjoude barat l CustomDiv l asese. ye3ne what i mean CustomDiv should have basicStyles in and also having as prop styles so the finalizedStyle will be {...basicStye,style}

Reply:
Done